caddyhttp/httpserver/server.go

```diff
@@ -507,16 +507,44 @@ func (s *Server) Stop() error {
 // OnStartupComplete lists the sites served by this server
 // and any relevant information, assuming caddy.Quiet == false.
 func (s *Server) OnStartupComplete() {
+
 	if caddy.Quiet {
 		return
 	}
+
+	s.OutputSiteInfo()
+
```

Comment:
Remove extra empty lines; these are very small, simple lines of code, so the extra lines look, well, empty. :)